setup.py

```diff
@@ -46,13 +46,30 @@ def fglob(patt):
         if not _.startswith("#") and "git+" not in _
     ]
 
+with open(os.path.join(rootdir, "requirements_excel.txt"), "r") as handle:
+    EXCEL = [
+        f"{_.strip()}"
```

Comment:
Avoid using `_` as variable name for a variable that is used, since the interpreter automatically assigns the last evaluation result to this variable. This can cause problems in interactive debugging. 

`line` is a much better name for this variable or just `s` for string.

Also `handle` is a really bad name for the return value of open(), which is a file object. . Hence, `f` is a well-recognised abbreviation for file objects.

Reply:
Hm.... I just copied this from the previous setup...